core/src/main/java/org/infinispan/cache/impl/EncoderCache.java

```diff
@@ -972,4 +1137,9 @@ public CompletableFuture<CacheEntry<K, V>> removeAsyncEntry(Object key) {
       K keyToStorage = keyToStorage(key);
       return cache.removeAsyncEntry(keyToStorage).thenApply(e -> unwrapCacheEntry(key, keyToStorage, e));
    }
+
+   @Override
+   public CompletionStage<Boolean> touch(Object key, boolean touchEvenIfExpired) {
```

Comment:
There is a `touch(Object, int, boolean)`, too.

Reply:
Done